packages/core/src/lib/implementation/read-rc-file.unit.test.ts

```diff
@@ -0,0 +1,92 @@
+import { vol } from 'memfs';
+import { join } from 'path';
+import { describe, expect } from 'vitest';
+import { CONFIG_FILE_NAME } from '@code-pushup/models';
+import { MEMFS_VOLUME } from '@code-pushup/testing-utils';
+import { readRc, readRcByPath } from './read-rc-file';
+
+describe('readRcByPath', () => {
+  it('should load a valid configuration file', async () => {
+    vol.fromJSON(
+      {
+        [`${CONFIG_FILE_NAME}.ts`]: '',
```

Comment:
It's confusing that an empty file is being used as a valid config example. Maybe a comment that the content is provided by `bundle-require` mock would help.

Reply:
added comments